# Generated by Django 3.0.7 on 2020-08-11 02:27

from django.db import migrations, models
import django.utils.timezone


class Migration(migrations.Migration):

    dependencies = [
        ('payment', '0002_auto_20200810_1646'),
    ]

    operations = [
        migrations.RemoveField(
            model_name='payment',
            name='number',
        ),
        migrations.AddField(
            model_name='payment',
            name='bank_unique_number',
            field=models.CharField(default=django.utils.timezone.now, max_length=300),
            preserve_default=False,
        ),
        migrations.AddField(
            model_name='payment',
            name='data_id',
            field=models.CharField(default=django.utils.timezone.now, max_length=300),
            preserve_default=False,
        ),
        migrations.AddField(
            model_name='payment',
            name='error_code',
            field=models.CharField(default=django.utils.timezone.now, max_length=100),
            preserve_default=False,
        ),
        migrations.AddField(
            model_name='payment',
            name='error_message',
            field=models.CharField(default=django.utils.timezone.now, max_length=500),
            preserve_default=False,
        ),
        migrations.AddField(
            model_name='payment',
            name='failed_at',
            field=models.DateTimeField(auto_now=True),
        ),
        migrations.AddField(
            model_name='payment',
            name='geo_unique_number',
            field=models.CharField(default=django.utils.timezone.now, max_length=300, unique=True),
            preserve_default=False,
        ),
        migrations.AlterField(
            model_name='payment',
            name='success_at',
            field=models.DateTimeField(auto_now=True),
        ),
    ]
